src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -56,12 +44,68 @@ void createStation() {
      * Then 2개의 지하철역을 응답 받는다
      */
     // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역 목록을 조회한다")
+    @Test
+    void listStation() {
+        // when
+        createStation("강남역");
+        createStation("신논현역");
+
+        // then
+        List<StationResponse> stations = getStations();
+
+        // then
+        assertThat(stations.stream().map(StationResponse::getName)).containsAll(Arrays.asList("강남역", "신논현역"));
+    }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
     // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    @DisplayName("전달받은 지하철역을 제거한다")
+    @Test
+    void deleteStation() {
+        // when
+        var station = createStation("역삼역");
+
+        // then
+        assertThat(station.getName()).isEqualTo("역삼역");
 
+        // then
+        List<StationResponse> beforeDelete = getStations();
+        assertThat(beforeDelete.size()).isEqualTo(1);
+        assertThat(beforeDelete.get(0).getName()).isEqualTo("역삼역");
+
+        // then
+        deleteStation(station);
+
+        // then
+        List<StationResponse> afterDelete = getStations();
+        assertThat(afterDelete.size()).isEqualTo(0);
+    }
+
+    private static StationResponse createStation(String name) {
+        return RestAssured.given().log().all()
+                .body(Map.of("name", name))
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().post("/stations")
+                .then().log().all()
+                .extract().jsonPath().getObject("", StationResponse.class);
```

Comment:
StationResponse로 변환하시려고 한거라면 요런 방법도 있습니다!
```suggestion
                .extract().response().as(StationResponse.class);
```

Reply:
좋네요!